docs/integrations/notification/index.rst

```diff
@@ -0,0 +1,164 @@
+########################################
+
+Monitoring experiment status is a vital part of working with Determined. In order to integrate
+Determined into your existing workflows, you can make use of webhooks to update other systems,
+receive emails, slack messages, and more when an experiment is updated.
+
+*****************************
+ Security and Signed Payload
+*****************************
+
+Each Webhook request will include a signed payload that users can use to verify that webhook calls
+from Determined are genuine. This will require that users define a ``webhook_signing_key`` for
+signing. If a webhook signing key is not provided then one will be generated for the user.
+
+Setting the Key
+===============
+
+The signing key can be set in the following ways:
+
+#. In the master yaml configuration file by adding:
+
+.. code::
+
+   security:
+       webhook_signing_key: <signing_key>
+
+2. By setting a ``DET_WEBHOOK_SIGNING_KEY`` environment variable.
+#. By specifying a ``---security-webhook-signing-key`` flag.
+
+Retrieving the Key
+==================
+
+The key can be found in the cluster configuration. For example it will be returned in
+``api/v1/master/config``.
+
+Event Payload
+=============
+
+Currently we will support two separate types of Webhooks ``Slack`` and ``Default``. A payload for a
+``Default`` webhook will contain information about the event itself, the trigger for the event, and
+the appropriate entity that triggered the event. Since we only support experiment state changes, all
+payloads will look like:
+
+.. code::
+
+   {
+      "event_id": "4cd26e62-60c6-4a8b-8d03-7629091a4ef5".   // unique event uuid,
+      "event_type": "EXPERIMENT_STATE_CHANGE" // the trigger type for the event,
+      "timestamp": 1665689991 // the timestamp of the event occurence,
+      "condition": {.
+         "state": "COMPLETED". // the condition that triggered the event for now the condition will always be of this format and contain either "COMPLETED" or "ERRORED"
+      },
+      "event_data": {
+         "experiment": {
+            "id": 41,
+            "state": "COMPLETED",
+            "name": "cifar10_pytorch_const",
+            "duration": 41, // this is in seconds
+            "resource_pool": "default",
+            "slots": 0,
+            "workspace": "test workspace", // name of the workspace
+            "project": "test project" // name of the project
+         }
+      }
+   }
+
+Signed Payload
+==============
+
+For every webhook request Determined will generate two headers ``X-Determined-AI-Signature``, and
+``X-Determined-AI-Signature-Timestamp`` which you can inspect to verify each request to their
+webhook endpoint.
+
+-  The ``X-Determined-AI-Signature-Timestamp`` will represent the time at which the request was
+   generated and sent. You can choose to inspect this timestamp and decide whether to discard any
+   requests with a timestamp that is too distant from the current time at their discretion.
```

Comment:
I would remove `at their discretion` here and make any changes needed to make the sentence readable.